### PR TITLE
Refactored disclosure to be generic

### PR DESCRIPTION
**Pull request by kingmushroom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +38,30 @@ disclosure.
...
+disclosure_column_2: The name of the column to put 'publish' marker. <br>
+disclosure_column_3: The name of the column to put reason for pass/fail. <br>
+parent_column: The name of the column holding the count of parent company.<br>
+threshold: The threshold above which a row is not disclosive.<br>
+
+### Stage 5
 
-##### Stage 5
+#### Runtime Variables Required:<br>
+disclosure_column_1: The name of the column to put 'disclosive' marker. <br>
```
Comment (thomashenson) [inline README.md]: As discussed with @kingmushroom & @Piwington these are set to be changed to have more meaningful names.

File: serverless.yml [hunk 2]
```diff
@@ -63,11 +63,6 @@ functions:
       - arn:aws:lambda:eu-west-2:${self:custom.accountId}:layer:es_python_marshmallow:2
     tags:
       app: results
-    environment:
-      disclosure_column_1: disclosive
```
Comment (thomashenson) [inline serverless.yml]: These names will need to change inline with the changes to the Python code.

Comment (Piwington) [reply serverless.yml]: Those variables don't exist Tom.

File: README.md (hunk 1 above)
Comment (Piwington) [reply README.md]: @thomashenson Pushed Variable Name Change.

**Pull request merged**